cmd/meroxa/root/apps/init.go

```diff
@@ -85,7 +85,7 @@ func (i *Init) Execute(ctx context.Context) error {
 
 	i.logger.Infof(ctx, "Initializing application %q in %q...", name, i.path)
 	switch lang {
-	case "go", GoLang:
+	case GoLang:
```

Comment:
fwiw, i kinda liked coercing the language value from the CLI into The Official Language in the backend.